src/screens/ContactScreen/elements/Search.js

```diff
@@ -102,7 +102,7 @@ const styles = StyleSheet.create({
     justifyContent: 'center'
   },
   newPostText: {
-    color: COLORS.holyTosca,
+    color: COLORS.holytosca,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been changed from `COLORS.holyTosca` to `COLORS.holytosca`. Please ensure that the new color `holytosca` exists in the `COLORS` object and it's not a typo. If it's a typo, here is the fix:
```diff
-    color: COLORS.holytosca,
+    color: COLORS.holyTosca,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->